xoto3/dynamodb/update/versioned.py

```diff
@@ -157,7 +160,7 @@ def versioned_diffed_update_item(
             update_item(table, item_key, **update_arguments, **expr)
             return updated_item
         except ClientError as ce:
-            if client_error_name(ce) == "ConditionalCheckFailedException":
+            if is_cancelled_and_retryable(ce):
```

Comment:
this is the actual change - we want to use a more robust method of determining whether this is retryable, and it just so happens we already have that function.